django_celery_results/backends/database.py

```diff
@@ -79,6 +79,10 @@ def _get_extended_properties(self, request, traceback):
                 # task protocol 1
                 task_kwargs = getattr(request, 'kwargs', None)
 
+            # TODO: We assuming that task protocol 1 could be always in use. :/
+            extended_props.update(
+                extend_task_props_callback(getattr(request, 'kwargs', None)))
```

Comment:
Consider calling this from `_store_result()` so it can be used regardless of the extended result setting.

Reply:
Nice point! It will be done in the next commit.